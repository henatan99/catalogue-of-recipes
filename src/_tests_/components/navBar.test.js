import React from 'react';
import { create } from 'react-test-renderer';

import NavBar from '../../components/navBar';

it('renders', () => {
  const tree = create(<NavBar />).toJSON;
  expect(tree).toMatchSnapshot();
});
