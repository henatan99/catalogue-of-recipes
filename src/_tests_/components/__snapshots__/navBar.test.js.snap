// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  1`] = `
<nav
  className="nav-bar"
>
  <ul
    className="nav-items"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav-item"
    >
      nomnom
    </li>
    <li
      className="nav-item"
    >
      <span
        className="iconify"
        data-icon="carbon:user-avatar-filled-alt"
        data-inline="false"
      />
    </li>
  </ul>
</nav>
`;
